zerver/actions/create_user.py

```diff
@@ -213,7 +213,7 @@ def process_new_human_user(
         and prereg_user.referred_by is not None
         and prereg_user.referred_by.is_active
     ):
-        # This is a cross-realm private message.
+        # This is a cross-realm direct message.
         with override_language(prereg_user.referred_by.default_language):
             internal_send_private_message(
```

Comment:
Here's a `private_message` reference, but maybe that's outside the scope of this PR?

Reply:
Yeah, my goal was for this pass to cover "private message" with a space, case-insensitive and also cover situations where there was a line break between those two words.